### DISCUSSION TITLE
WARNING: precursors were wrongly annotated in the library as proteotypic

### DISCUSSION THREAD

**Discussion by proteollyx** [vdemichev/DiaNN#629] [Q&A]
Dear Vadim, dear DIA-NN community,

I'm running DIA-nn (1.8.1) using a previously created predicted spectral library (from fasta generated with DIA-NN). I run the analysis from Thermo DIA raw files (non pre-existing .quant files) and I get the following error message for each raw file:

**WARNING: 63408 precursors were wrongly annotated in the library as proteotypic**

What is the reason for this error?
Should I be worried about that?

Thank you for you insights!
Best
Oliver

**Comment by proteollyx** [comment]
I played around with a subset of my data and could fix the issue by adding the fasta files on top and apply _reannotate_.

**Comment by vdemichev** [comment]
Hi Oliver,

This will happen if a different FASTA is used or protein inference setting is changed.

Best,
Vadim
